### ISSUE TITLE
[UX] Do not redirect to the menu listing page when creating a new menu

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#4486]
**Steps To Reproduce**

1. Navigate to Structure -> Menus -> Add menu (`admin/structure/menu/add`)
2. Add a title for the menu, and click "Save" ...

**Actual behavior**

You are redirected to the menu listing page, and now have to click the "Manage links" button, and then click "+ Add link" ...that's too many clicks away from the task that most will need to do 👎 (I mean, who wants to create a menu, and then leave it at that)

**Expected behavior**

You are either redirected to the "Manage links" page for the newly-created menu, or we have 2 submit buttons:
- Save and add menu links
- Save & exit (a-la Views)

...while at it, add "Cancel" links to the various menu and menu link related forms.

**Comment by indigoxela** [comment]
@klonos Many thanks for taking this over. Yes, that seems to be yet another "Drupalism". :wink: 

How about simplifying menu forms? For instance by merging the "List links" and "Edit menu" operations into a single administration page.

There's an (unreleased) contrib module for that: https://github.com/backdrop-contrib/simplified_menu_admin

Or do you think this is out of the scope of this issue?

**Comment by klonos** [comment]
Yeah, that's a different issue @indigoxela ...this one here is a UX bug, whereas what you suggest is a new feature ...and we already have an (old) issue for it: #475 😄
